packages/ui/react/src/components/Templates/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from '@storybook/react'
```

Comment:
@thlmenezes could you please ensure a minimal component description for each component?

For example, when accessing `Button > Docs` we see the text 
```
Primary UI component for user interaction
```

However, when accessing Primitives, Templates, Tooltips, Tab, etc, I couldn't understand their purpose.

I believe a short text in each componente would be helpful.

<img width="789" alt="image" src="https://github.com/user-attachments/assets/9a3bd9de-649c-44d8-8cae-a02691123061">

Reply:
> I believe a short text in each componente would be helpful.

Agreed, added some texts to describe both components and a new deployment is on the way https://app.devopness.com/actions/40235

Reply:
![image](https://github.com/user-attachments/assets/b3a9aa1b-383a-48ba-8c4c-caeae524d627)

![image](https://github.com/user-attachments/assets/5b609f96-8700-4e87-a3ca-0543c2252b89)